data/shared/citizen/scripting/v8/main.js

```diff
@@ -297,6 +302,7 @@ const EXT_LOCALFUNCREF = 11;
 		return frames;
 	}
 	
+	Error.stackTraceLimit = 16;
```

Comment:
is this related to the PR's purpose?

Reply:
The default is 10.  
Not enough in many cases.